Sources/Remote Logging/Crash Logging/ApplicationFacade.swift

```diff
@@ -37,6 +44,31 @@ final class ApplicationFacade {
 
 #if os(iOS)
 
+private extension UIApplication {
+    // When compiling with Swift Package Manager, it wants us to
+    // use only extension-safe API. We still want to use UIApplication
+    // when it's available, while not using extension-unsafe API
+    // when not available. So we're going to be sneaky about getting
+    // `UIApplication.shared`, but only when it should already be safe.
```

Comment:
At least in Xcode 13 b3, SPM packages are compiled in extension-safe mode, which prohibits calling `UIApplication.shared`. This change may be reverted in later Xcode betas, as there has been a lot of negative feedback on the topic, but this code should be safe either way.